libraries/AP_GPS/AP_GPS_GSOF.h

```diff
@@ -48,10 +48,10 @@ class AP_GPS_GSOF : public AP_GPS_Backend
     uint32_t SwapUint32(uint8_t* src, uint32_t pos);
     uint16_t SwapUint16(uint8_t* src, uint32_t pos);
 
-
     struct gsof_msg_parser_t
     {
-        enum
+
+        enum class gsof_states
```

Comment:
opening brace should go on this line

Reply:
I was told to preserve formatting, even if it doesn't comply with the style guide. 

If you want me to fix the formatting in this driver, can I do it in a separate PR? This PR is for enum classes, not formatting.

Reply:
I prefer fixing the formatting in the code as you're making legitimate changes to the code, rather than "whitespace only" changes.

But Sure, if you don't want to change this, don't change it,

Reply:
I'll do it in another, once the other 4 PR's go in, otherwise it's merge conflict madness.